src/Entities/Boat.h

```diff
@@ -31,8 +31,27 @@ class cBoat :
 	virtual void HandleSpeedFromAttachee(float a_Forward, float a_Sideways) override;
 
 	cBoat(double a_X, double a_Y, double a_Z);
-} ;
 
+	int GetTimeSinceLastHit(void) const { return m_TimeSinceLastHit; }
```

Comment:
This is one of the variables specified in the boat metadata. I'm not sure where exactly it's used or why they've named it as such, as I couldn't find any protocols that pass it other than when the metadata is being passed to other clients for boats.

I'll look a bit more into it and see if I can figure out its purpose.